shuup/gdpr/models.py

```diff
@@ -29,6 +29,11 @@ class GDPRSettings(TranslatableModel):
         verbose_name=_('enabled'),
         help_text=_("Define if the GDPR is active.")
     )
+    skip_consent_on_auth = models.BooleanField(
+        default=False,
+        verbose_name=_('skip_consent_on_login'),
```

Comment:
oh make this label prettier 
![image](https://user-images.githubusercontent.com/8770370/58251920-512da880-7d3b-11e9-8644-707d6eb76c32.png)